truffle/LoadContract.html

```diff
@@ -157,15 +182,52 @@
 end
 
 loop While in Progress
-C ->> L: setVaultHash()
-activate L
-L -->> SL: setVaultHash(hash)
-activate SL
-deactivate SL
-note over L: emit VaultHash
-deactivate L
+C ->> V: setVaultHash(vaultId, vaultHash)
+activate V
+note over V: emit VaultHash
+deactivate V
 end
 
+opt Need to Revoke the Permission of Carrier
```

Comment:
I still don't think it's necessary to document the carrier revocation separately from the other ACL calls

Reply:
The thing concerned me is the "opt" and "alt" outside the general calls. It may make the reader think granting the update permission of hash for the carrier can be optional. But, it is actually not. That's why I did not put opt box for grantUpdateHashPermission(vaultId, Carrier), and only for the revokeUpdateHashPermission(vaultId, Carrier). That's based on my understanding. not sure make sense or not?

Reply:
I think it's fine to leave `grantUpdateHashPermission(vaultId, Carrier)`, but remove `revokeUpdateHashPermission(vaultId, Carrier)` altogether because it's probably going to be a pretty rare use-case, and it's already technically covered under the section that shows all the ACL calls.